java/code/src/com/redhat/rhn/frontend/xmlrpc/image/profile/ImageProfileHandler.java

```diff
@@ -116,10 +117,11 @@ public ImageProfile getDetails(User loggedInUser, String label) {
      * @xmlrpc.param #param("string", "storeLabel")
      * @xmlrpc.param #param("string", "path")
      * @xmlrpc.param #param_desc("string", "activationKey", "Optional")
+     * @xmlrpc.param #param_desc("string", "kiwiOptions", "Optional")
      * @xmlrpc.returntype #return_int_success()
      */
     public int create(User loggedInUser, String label, String type, String storeLabel,
-            String path, String activationKey) {
+            String path, String activationKey, String kiwiOptions) {
```

Comment:
This does not seems really optional to me.
How will this behave when you call create from xmlrpc and omit `kiwiOptions` option?

Seems to me, because of all the changed unit tests that it will not work -> breaking change in API.
I would rather create new overloaded create function with added kiwiOptions param.